README.md

```diff
@@ -1 +1,47 @@
-# New-York-Times-Clone
\ No newline at end of file
+# New York Times Clone Page
+
+This assignment consists of making a clone of a New York Times article page. By this point, the student should be familiar with the concepts of:
+
+Embedding images
+Working with forms (buttons in this case)
+Floats and positioning
+The box model
+
+![screenshot](./app_screenshot.png)
+
+## Built With
+
+- HTML/CSS,
+- VISUAL STUDIO CODE
+- GIT
+- Lynters
+
+## Live Demo
+
+[Live Demo Link](https://raw.githack.com/rgcavalheiro/New-York-Times-Clone/nyt-article-page/index.html)
+
+
+
```

Comment:
- [ ] You should add a section on how to open the project locally. It is a good practice to do this from here onwards.